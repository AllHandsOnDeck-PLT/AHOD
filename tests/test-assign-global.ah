int x
bool y 
string z
main:
  {
  x = 4
  do PRINT(x)
  y = true
  do PRINT(y)
  z = "banana"
  do PRINT(z)
  }
  