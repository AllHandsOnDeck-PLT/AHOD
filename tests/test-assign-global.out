4
1
banana
